src/Secret.php

```diff
@@ -125,26 +152,27 @@ public function getSecrets(string $nextLink = null): IdRepository {
 
     /**
      * Sets a secret in a specified key vault.
-	 * If the named secret already exists, Azure Key Vault creates a new version of that secret.
+     * If the named secret already exists, Azure Key Vault creates a new version of that secret.
      * @param string $secretName
      * @param string $value
      * @param SecretAttributeEntity|null $secretAttributes
      * @param string|null $contentType
      * @param array|null $tags
      * @return SecretEntity
      */
-    public function setSecret(string $secretName, string $value, $secretAttributes = null, string $contentType = null, array $tags = null) {
+    public function setSecret(string $secretName, string $value, $secretAttributes = null, string $contentType = null, array $tags = null)
```

Comment:
What secrets do we need to set programmatically like this?

Reply:
None, but this is forked from a library that included all of this logic. So there's several methods that we'll probably never use, but it's not worth going through and removing them all.

Reply:
What do you think about removing the set specifically then? That way we don't have to give the site write permissions in keyvault and reduce our attack surface.

Reply:
On second thought we could just not give the applications write access and it doesn't matter if we have this code or not.